simple_form.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "simple_form"
 
-  s.add_dependency('activemodel', '> 4', '< 5.2')
-  s.add_dependency('actionpack', '> 4', '< 5.2')
+  s.add_dependency('activemodel', '> 4', '< 6')
```

Comment:
Let's remove the upper bound. Let it break, with Rails 6.0 or maybe it will not as we can see it happen with Rails 5.2, but that will not require us to make a new release just to change the upper bound in case it works out of the box.